network-api/networkapi/wagtailpages/templates/wagtailpages/opportunity_page.html

```diff
@@ -25,12 +25,22 @@ <h1 class="h1-heading {% if uses_menu %}d-none d-md-block{% endif %}">
         {{ page.title }}
       {% endif %}
     </h1>
-    <div class="cms">
+    <div class="cms oppurtunity-page-content">
```

Comment:
Typo. `opportunity`

Reply:
Thank you for the catch Kalob! Updated 👍